bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueLightHandler.java

```diff
@@ -440,15 +443,28 @@ private StateUpdate createBrightnessStateUpdate(int currentBrightness, int newBr
     }
 
     @Override
-    public void onLightStateChanged(@Nullable HueBridge bridge, FullLight fullLight) {
+    public boolean onLightStateChanged(@Nullable HueBridge bridge, FullLight fullLight) {
         logger.trace("onLightStateChanged() was called");
 
+        // TODO Maybe this check is obsolete
```

Comment:
With your changes, this check becomes useless (it was useful before).

Reply:
Removed